modules/oc-mirror-image-set-config-examples.adoc

```diff
@@ -249,53 +249,30 @@ mirror:
 ----
 
 [discrete]
-[id="oc-mirror-image-set-examples-blocked-images_{context}"]
+[id="oc-mirror-image-set-examples-mce-images-catalog-ocp_{context}"]
 == Use case: Including the `blockedImages` parameters
 
-The following `ImageSetConfiguration` file excludes the images that return `manifest unknown` errors.
-
-This example also shows how to use the `blockedImages` parameter in the `imageSetConfiguration` file to skip mirroring an image that fails to mirror due to an `manifest unknown` error.
+The following `ImageSetConfiguration` file includes the {mce} Operator and all {product-title} versions starting at a minimum version of `4.17.0` to `4.17.1`  in the channel. 
```

Comment:
This complete change is not looking proper. Better to discuss once on this.

Reply:
Expectation is to have new content under : "Use case: Including the multi -arch OpenShift Container Platform images and catalog for multicluster-engine operator. "

Reply:
Not sure why we are removing the existing content for amd64